### DISCUSSION TITLE
Dual Tabs in the same browser window

### DISCUSSION THREAD

**Discussion by Daszbin** [cefsharp/CefSharp#3604] [General]
Please ask your questions under the Discussions section here on GitHub 
- https://github.com/cefsharp/CefSharp/discussions

---

`CefSharp` is a wrapper around the [Chromium Embedded Framework](https://bitbucket.org/chromiumembedded/cef/overview) in a lot of cases a feature must be added to `CEF` first before it can be used in `CefSharp`. `CEF` has its own `Feature Request Forum` at https://magpcss.org/ceforum/viewforum.php?f=7

--- 

**Describe the solution you'd like**
A clear and concise description of the feature you are proposing
There is a feature in a chromium browser called Naver Whale it’s a South Korean web browser using the same engine they have dual browser. With dual tab it open two tab in the same browser window. It also can merge two opened tabs into one dual tab screen.

**Describe alternatives you've considered**
I haven’t tried anything yet I wanted to see if it could be implemented some how.
**Additional context**
![79B49DE6-903E-4F08-A4FB-12D6AA7FBE55](https://user-images.githubusercontent.com/37669537/120908296-b7da4200-c636-11eb-8716-25ea0a669ae3.jpeg)
![C8BFCA96-F93C-4553-A1F8-7729AE312A7C](https://user-images.githubusercontent.com/37669537/120908297-b9a40580-c636-11eb-8ef3-99f6bd0ad2ad.jpeg)
![8EB08952-79E2-4A73-B994-22AF86C9CF49](https://user-images.githubusercontent.com/37669537/120908298-ba3c9c00-c636-11eb-834e-09cab233e3bb.jpeg)
![E0C23ABF-BEB7-4A62-A572-9A51078F63B8](https://user-images.githubusercontent.com/37669537/120908299-bad53280-c636-11eb-967b-f239d7c0a4c2.jpeg)
![04631946-EE37-40CF-AA40-936817F19DBB](https://user-images.githubusercontent.com/37669537/120908301-bb6dc900-c636-11eb-951d-2f034e5d6523.jpeg)

**Checklist:**
- [x] I have reviewed the [CEF API](https://magpcss.org/ceforum/apidocs3/index-all.html) and have confirmed the feature I'm requesting is possible.
- [x] The Feature I'm requesting is an improvement to Async Javascript Binding (No new feature are being added to the Sync Javascript binding)
- [x] An open `PR` exists on the [CEF Pull Requests](https://bitbucket.org/chromiumembedded/cef/pull-requests/) and I'd like to propose this feature is added to `CefSharp` when/if it's merged.

**Comment by amaitland** [comment]
This functionality is up to you to implement as there is no tab functionality included. Add multiple ChromiumWebBrowser instances to a single tab, move them between tabs, they are just controls. 

In future please use https://github.com/cefsharp/CefSharp/discussions rather than open a feature request.
